skimage/color/colorconv.py

```diff
@@ -1513,8 +1519,6 @@ def combine_stains(stains, conv_matrix):
     >>> ihc_hdx = separate_stains(ihc, hdx_from_rgb)
     >>> ihc_rgb = combine_stains(ihc_hdx, rgb_from_hdx)
     """
-    from ..exposure import rescale_intensity
```

Comment:
I think this was avoiding a circular import.

Maybe best to leave this to a separate PR.

Reply:
You got it.